lib/scientist/experiment.rb

```diff
@@ -269,4 +269,16 @@ def try(name = nil, &block)
   def use(&block)
     try "control", &block
   end
+
+  def raise_on_mismatches=(bool)
```

Comment:
I might go with `attr_accessor` here instead, but continue to define the predicate method below.